docs/tags/index.md

```diff
@@ -29,6 +29,38 @@ Containers, Kubernetes clusters and cloud environments regularly start and termi
   <source src="https://cdn.sematext.com/videos/groupbytags2.mp4" type="video/mp4" />
 </video>
 
+### Benefits of using Tags ###
+Some benefits of using tags are:
+
+#### Better organization ####
+Tags provide a flexible way to organize logs and metrics into logical groups. This can be especially useful when dealing with large amounts of data, as it can help users to quickly and easily find the information they need. For example, a tag could be used to group logs from different components of an application or to group metrics by geographic region.
+
+#### Easier filtering ####
+Tags allow users to filter data based on specific criteria. For example, users can filter logs or metrics by tag to quickly see all the data associated with a particular component or service. This makes it easier to find and troubleshoot issues. Filtering by tag is also useful when [creating alerts](https://sematext.com/docs/alerts/), as users can set up alert rules based on specific tags.
+
+#### Improved searchability ####
+Tags provide additional context for logs and metrics, which makes them more searchable and discoverable. Users can search for logs or metrics based on tag values to find related data, even if the data is spread across multiple sources. This can be especially useful when investigating issues, as it can help users to quickly find all the relevant data in one place.
+
+#### Customizable dashboards and reports ####
+Tags can be used to create custom [dashboards](https://sematext.com/docs/dashboards/) and [reports](https://sematext.com/docs/dashboards/reports-and-components/) that display data specific to certain tags. For example, a dashboard could be created to display specific application metrics, making it easier to monitor and optimize performance. Dashboards can be [shared](https://sematext.com/docs/team/account-members/) with others, providing a consistent view of metrics across a team or organization.
+
+In summary, tags provide a powerful way to organize, filter and search data. They can be used to provide additional context and metadata, making it easier to navigate and analyze large amounts of data.
+
+### Common ways to use Tags ###
+Here are some examples of the most common ways to use tags in a Kubernetes cluster:
```

Comment:
@cpipilas Suggest you drop "Kubernetes" and tweak any k8s-specific stuff below (e.g. pods) because all of this is applicable to non-k8s environments, too.

Reply:
Content updated